tendrl_node_agent/monitoring/plugins/handle_collectd_notification.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+import sys
+from subprocess import check_output
+import socket
+import json
+
+def getNotification():
+    notification_dict = {}
+    isEndOfDictionary = False
+    for line in sys.stdin:
+        if not line.strip():
+            isEndOfDictionary = True
+            continue
+        if isEndOfDictionary:
+            break
+        key, value = line.split(':')
+        notification_dict[key] = value.lstrip()[:-1]
+    return notification_dict, line
+
+
+def postNotificationToNodeAgentSocket():
+    s = socket.socket()
+    host = "127.0.0.1"
```

Comment:
I didn't want it to be configurable unless there's a strong need for it. Because, its something internal to collectd-to-node-agent communication. But, yes if needs to be configurable then collectd side our custom plugin would need to read tendrl's conf file because it might not be good to replicate this config value in both collectd's conf file and also in tendrl's conf file.